src/com/jeopardy/AdvancedPlayer.java

```diff
@@ -16,18 +15,17 @@ public int deductScore(int currentQuestionValue) {
     return score;
   }
 
-  @Override
+  @Override  //implement interface abstract method
   public int askForHelp(int currentQuestionValue) {
     int result = 0;
-    System.out.println("Help has been called and you are getting nothing for this one!");
```

Comment:
Let's keep your prinline message here. I don't know how it got overriden by my old message.